src/main/java/ghostcat/capstone/image_query/ImageQueryDAO.java

```diff
@@ -32,6 +28,21 @@
         Table userDataTable = dynamoDB.getTable(PROJECT_TABLE);
         ArrayList<Item> results = new ArrayList<>();
 
+        AmazonDynamoDB c = AmazonDynamoDBClientBuilder.standard()
+                .withRegion(Regions.US_EAST_2)
+                .build();
+        DynamoDB d = new DynamoDB(c);
+
+        TableCollection<ListTablesResult> tables = dynamoDB.listTables();
+        Iterator<Table> it = tables.iterator();
+
+        System.out.println("Listing table names");
+
+        while (it.hasNext()) {
+            Table table = it.next();
+            System.out.println(table.getTableName());
+        }
```

Comment:
Do we still need this Mattie? It looks like a bit of a test, might be best to get rid of this now